charts/agent/templates/daemonset.yaml

```diff
@@ -175,6 +175,9 @@ spec:
               name: modprobe-d
               readOnly: true
             {{- end }}
+            - mountPath: /host/etc
```

Comment:
with this addition I think the `osrel` and `etc-fs` mounts/volumes can be removed